tasks/query-cmr/app/cli.ts

```diff
@@ -68,8 +67,8 @@ export default async function main(args: string[]): Promise<void> {
   const startTime = new Date().getTime();
   const appLogger = logger.child({ application: 'cmr-granule-locator' });
   const options = parser().parse(args);
-  const encrypter = createEncrypter(env.sharedSecretKey);
-  const decrypter = createDecrypter(env.sharedSecretKey);
+  const encrypter = createEncrypter(process.env.SHARED_SECRET_KEY);
```

Comment:
Is the intent to make query-cmr independent from the .env config mechanism?  I think that seems right.

Reply:
I made these changes thinking that the main harmony .env file was specific to the harmony code (especially with the typescript variable declarations) and that we should just use environment variables directly in query-cmr. It turns out that other namespaces used from the main harmony code pull in the .env file anyway, so this change wasn't necessary in the end, but I still thought it was a better idea.